src/RestApi/Controllers/ProductCategories.php

```diff
@@ -158,14 +197,20 @@ public function get_item_schema() {
 			'properties' => array(),
 		);
 
-		$schema['properties']['id']          = $raw_schema['properties']['id'];
-		$schema['properties']['name']        = $raw_schema['properties']['name'];
-		$schema['properties']['slug']        = $raw_schema['properties']['slug'];
-		$schema['properties']['parent']      = $raw_schema['properties']['parent'];
-		$schema['properties']['count']       = $raw_schema['properties']['count'];
-		$schema['properties']['description'] = $raw_schema['properties']['description'];
-		$schema['properties']['image']       = $raw_schema['properties']['image'];
-		$schema['properties']['permalink']   = array(
+		$schema['properties']['id']           = $raw_schema['properties']['id'];
+		$schema['properties']['name']         = $raw_schema['properties']['name'];
+		$schema['properties']['slug']         = $raw_schema['properties']['slug'];
+		$schema['properties']['parent']       = $raw_schema['properties']['parent'];
+		$schema['properties']['count']        = $raw_schema['properties']['count'];
+		$schema['properties']['description']  = $raw_schema['properties']['description'];
+		$schema['properties']['image']        = $raw_schema['properties']['image'];
+		$schema['properties']['review_count'] = array(
+			'description' => __( 'Number of reviews in the category.', 'woo-gutenberg-products-block' ),
```

Comment:
Just needs a note here about the collection parameter because this isn't always set.